lib/sinon/util/fake_server.js

```diff
@@ -137,13 +142,21 @@ var fakeServer = {
         }
     },
 
+    logger: function () {
+        // no-op; override via configure()
```

Comment:
I do not see how this is overridden by the configure call. `this.logger` has only read-references in this file, and the `this` object is not passed on to the `configure` method of `./core/log_error`. Therefore the check in line 157 makes no sense as it will always be true. What am I missing here?